docs/pages/kubernetes-access/helm/reference.mdx

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
We're no longer keeping Helm references in the Kubernetes Access section, so we should remove this page.